module-4/backend/week_one.md

```diff
@@ -5,15 +5,44 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+getting used to the new syntax
+
 2. What are some tools to help debug JavaScript code?
+
+dedugger, console.log
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+mocha, chai. ive heard of jasmine too but i havent tried it out yet
+
 4. What is the syntax for invoking a function? Give an example.
+
+function()
+
 5. What's `this` in JavaScript?
+
+what are currently in. like an object or an event. when you click on something, this is now what you are inside of
```

Comment:
This is the kind of question I'd go into more detail in so you ensure you fully/clearly communicate your understanding.